lib/modules/datasource/conan/index.ts

```diff
@@ -62,10 +106,12 @@ export class ConanDatasource extends Datasource {
     packageName,
   }: GetReleasesConfig): Promise<ReleaseResult | null> {
     const depName = packageName.split('/')[0];
-    const userAndChannel = '@' + packageName.split('@')[1];
+    const userAndChannel = '@' + packageName.split('@')[1].split('#')[0];
+    const revision = getRevision(packageName);
```

Comment:
revert, `GetReleaseConfig` should not handle the new digest. You need to override the `GetDigest` function as told before.

`currentDigest` is a property of `GetReleasesConfig` interface

Reply:
I did override the `getDigest` function, however I also kept the implementation in order to have digest updates and new versions. 
I tried to remove it from `getReleases but renovate created a new pull request like this:
```diff
- cairo/1.17.2@_/_#aff2d03608351db075ec1348a3afc9ff
+ cairo/1.17.2@_/_#51627b399b12479e23cc7958a33fb5eb
```
which makes no sense since this is a revision for `cairo/1.17.4` and not `cairo/1.17.2` . Or maybe I am missing on how I can provide multiple digests. 

`currentDigest` is not a property of GetReleasesConfig: 
https://github.com/renovatebot/renovate/blob/dd5ca94a03cdfb5d3db21e66ba0d4f7de2500786/lib/modules/datasource/types.ts#L21-L24.